packages/wrangler/src/__tests__/https-options.test.ts

```diff
@@ -84,25 +85,57 @@ describe("getHttpsOptions()", () => {
 		expect(std.err).toMatchInlineSnapshot(`""`);
 	});
 
+	it("should warn if not able to write to the cache (legacy config path)", async () => {
```

Comment:
what does "legacy" mean here?

Reply:
It is referring to the paths to the `os` home directory which is still checked for so we needed a test to confirm that functionality worked as well and was distinguishable from XDG compliant paths.